Arduino-Bling/RobotBling/animations.ino

```diff
@@ -40,3 +40,33 @@ void blinkOffWithDelay(uint8_t wait)
   setAllPixelsOff();
   delay(wait);
 }
+
+ void strobe(rgbColor c, uint8_t numFlashes, int flashDelay, int strobePause)
+ {
+  for(int j = 0; j < numFlashes; j++) 
```

Comment:
Whitespace is a good thing in coding and life :) Unlike function calls (such as `delay()`), when using conditional constructs, ie. `if statement`, and `loops like for`, please add space before the left `(`. Something like:
`for (int j...)`